src/app/components-new/authentication/MnemonicInput.vue

```diff
@@ -61,6 +61,7 @@ div
                 const value = this.mnemonicFields
                     .map(field => field.value.trim().toLocaleLowerCase())
                     .join(' ')
+                    .trim()
```

Comment:
Why is this necessary, BTW? I can't tell by looking at the code.

Reply:
To remove a trailing whitespace at the end of the mnemonic if all the fields aren't filled in